### PR TITLE
Social media sharing links

### PR DESCRIPTION
**Pull request by AsmahanM**
- [x] Added links to share pages on social media (facebook, whatsapp, twitter & email.)

At the moment, the URL is being copied and the user is redirected to the relevant platform but a *clickable* link isn't being generated, just a string of the URL.

### CODE REVIEW

**Comment by sonianb** [comment]
[Add share button](https://trello.com/c/3tYoD78p/34-add-share-button)

**Review by sonianb — APPROVED** [review]
Great work, really like this feature ✨

File: components/ShareLink/SocialShare.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    
+      useEffect(() => {
+        setURL(window.location.href)
+      }, [])
+
+
+    return (
+        <div>
+            <p>Share with your friends!</p>
```
Comment (alternadiva) [inline components/ShareLink/SocialShare.js:28]: So exciting 🌞 can't wait to try

**Review by alternadiva — APPROVED** [review]

**Review by minju25kim — APPROVED** [review]

**Comment by sonianb** [comment]
[When group is created, user will be able to share a group invite on different platforms via the share link.](https://trello.com/c/y1th8Zti/7-when-group-is-created-user-will-be-able-to-share-a-group-invite-on-different-platforms-via-the-share-link)

**Pull request merged**